sdks/wasm/incrementalrunner.cs

```diff
@@ -97,7 +97,13 @@ public TestAction (TestMethod test, AbstractAction parent, IncrementalTestRunner
 		SetupContext ();
         try
         {
-            testResult = test.MakeTestCommand ().Execute(this.context);
+			if (IncrementalTestRunner.testSkipCount > 0) {
+				--IncrementalTestRunner.testSkipCount;
+				testResult = ((Test)test).MakeTestResult ();
+				testResult.SetResult (ResultState.Success);
+			} else {
+				testResult = test.MakeTestCommand ().Execute(this.context);
+			}
         }
         finally
         {
```

Comment:
nit: the try-finally blocks use spaces instead of tabs.

Reply:
fixed